src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/JSON_PowerFxV1CompatibilityRules.txt

```diff
@@ -0,0 +1,87 @@
+#SETUP: PowerFxV1CompatibilityRules
+#SETUP: EnableJsonFunctions
+
+>> JSON(ParseJSON("{}"))
```

Comment:
Most of those tests will work in Power Apps both with and without PFx1.0 enabled, so they can go in the "base" JSON.txt test file. The only ones which require V1 are the ones with decimal, and the ones that rely on the "untyped number" logic, where numbers are stored as raw values (strings) until needed.